src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
 
-* *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+** *Thumbnail* - Displays how your Koji will look if it's shared on social media.
 
-* *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+To learn more about using previews, see <<testing-templates#_testing_in_the_koji_editor,Testing in the Koji editor>>.
 
-* *Sticker* - Displays how your template will look if it's imbedded in another template.
 
-* *Thumbnail* - Displays how your template will look if it's shared on social media.
+=== Previews outside the Code Editor
 
-=== Remote previews
+You can also preview a Koji outside the Code Editor.
 
-The preview pane provides these options for previewing a template outside of the Koji editor.
+* The QR code button image:{imagesDir}/QRcode.svg[QR code button] displays a QR code for your Koji.
+Scan this code to preview the Koji on your mobile device.
 
-* The QR code button image:QRcode.png[QR code button] displays a QR code for the template.
-You can scan this code to preview the template on your mobile device.
+* The remote staging button image:{imagesDir}/remote-staging.svg[Remote staging button] opens a preview of your Koji in a new browser tab.
 
-* The remote staging button image:remote-staging.png[Remote staging button] opens a preview of your template in a new browser tab.
 
-=== Launching the debugger
+== Debugging
 
-*Launch Debugger* opens the debugger in a new browser tab.
+To open the debugger in a new browser tab, click *Launch Debugger*.
 
 
-[.faq]
 == Tips
-[id="faq-nodejsnpmver"]
-.What versions of Node.js and npm do Kojis run on?
+
+* While editing a plain text file, such as .html or .json, you can open menus with additional editing options.
++
+.Learn how...
 [%collapsible]
 ====
-include::{includespath}/misc.adoc[tag=nodejsnpmver]
+[.singlespaced]
+. In the Code Editor, move the focus to the editing pane where the contents of the text file is displayed.
+. Right-click anywhere in the editing pane for the contextual menu.
+. Press *F1* for the Command Palette menu.
 ====
 
-[id="faq-revert-kojiversion"]
-.How do I revert to an old version of my template?
+* You can revert to an old version of your project.
++
+.Learn how...
 [%collapsible]
 ====
-To revert to a previous commit,
-
-include::{includespath}/steps-codeeditor.adoc[tag=openproject-template2edit]
-
-include::{includespath}/steps-codeeditor.adoc[tag=expand-terminal]
-
+[.singlespaced]
+. Open your project in the Code Editor.
+. Expand the terminal pane.
 . Run `git log` and copy the hash code of the commit you want to restore.
-
-. In the terminal panel, enter `git reset --hard <commit-hash>`.
+. In the terminal pane, enter `git reset --hard <commit-hash>`.
 +
 WARNING: You will lose all the changes you made after the commit you restore.
```

Comment:
Consider moving this to the topic of the procedure so they read it before they do something destructive to their code.

Reply:
Moved and rewrote for clarity.